src/EFTools/EntityDesignerVersioningFacade/Resources.VersioningFacade.resx

```diff
@@ -207,4 +207,7 @@
   <data name="EF6Providers_NotDbProviderServices" xml:space="preserve">
     <value>The 'Instance' member of the Entity Framework provider type '{0}' did not return an object that inherits from 'System.Data.Entity.Core.Common.DbProviderServices'. Entity Framework providers must extend from this class and the 'Instance' member must return the Singleton instance of the provider.</value>
   </data>
+  <data name="CoercingNullablePrimaryKeyPropertyToNonNullable" xml:space="preserve">
+    <value>Primary key column {0} on table {1} is nullable. Entity Framework does not allow non-null primary keys so it will coerce the matching primary key property {2} to be non-nullable.</value>
```

Comment:
I like the comment "any null values stored in this column could cause errors at run time" as that's useful for the user to know, but I like the message to say what it is doing about the situation. @divega's suggested message does not say that we are making the generated properties non-nullable which I think we should mention.